components/identity.oauth.uma.dbscripts/uma_tables.sql

```diff
@@ -1,17 +1,18 @@
 CREATE TABLE IDN_RESOURCE (
-  ID                INTEGER   AUTO_INCREMENT PRIMARY KEY NOT NULL,
-  RESOURCE_ID       VARCHAR(255),
-  RESOURCE_NAME     VARCHAR(30),
-  TIME_CREATED      TIMESTAMP DEFAULT '0',
-  RESOURCE_OWNER_ID VARCHAR(255),
-  TENANT_ID         INTEGER
+  ID                  INTEGER   AUTO_INCREMENT PRIMARY KEY NOT NULL,
+  RESOURCE_ID         VARCHAR(255),
+  RESOURCE_NAME       VARCHAR(255),
+  TIME_CREATED        TIMESTAMP DEFAULT '0',
+  RESOURCE_OWNER_NAME VARCHAR(255),
+  CONSUMER_KEY        VARCHAR(255),
```

Comment:
Let's use client_id instead of consumer_key

Reply:
fixed: https://github.com/wso2-extensions/identity-oauth-uma/blob/637fc5618e6fb2e66ad1ee3238291cebe2ca18f8/features/org.wso2.carbon.identity.oauth.uma.server.feature/resources/dbscripts/h2.sql#L7